exporter/debugexporter/exporter.go

```diff
@@ -95,3 +101,20 @@ func (s *debugExporter) pushLogs(_ context.Context, ld plog.Logs) error {
 	s.logger.Info(string(buf))
 	return nil
 }
+
+func (s *debugExporter) pushProfiles(_ context.Context, pd pprofile.Profiles) error {
+	s.logger.Info("Profiles",
+		zap.Int("resource profiles", pd.ResourceProfiles().Len()),
+		zap.Int("sample records", pd.SampleCount()))
+
+	if s.verbosity == configtelemetry.LevelBasic {
+		return nil
+	}
+
+	buf, err := s.profilesMarshaler.MarshalProfiles(pd)
+	if err != nil {
+		return err
+	}
+	s.logger.Info(string(buf))
```

Comment:
Add test for this?

Reply:
Hey @mx-psi the test actually follows the same structure for the other signals.
I've added 2 tests in the `exporter_test`:
- `TestErrors`
- `TestProfilesNoErrors`

Those are exactly the same tests all other signals have.

Let me know if you'd like me to add anything different for Profiles.

Reply:
Makes sense, thanks. I think we should increase coverage (and test that we are actually logging something), but we don't need to do it now, we can do it together with all signals